### PR TITLE
Api call navbar

### PR DESCRIPTION
**Pull request by phoenixAja**
### Description

integrate usergroup api with navbar component

#### Issue
[ch119785](https://app.clubhouse.io/genepi/stories/space/119785)

### Test plan

refreshed page when logged in, saw user name and dph name values change

### CODE REVIEW

File: src/py/aspen/app.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def decorated(*args, **kwargs):
 # else send index.html. Allows reloading React app from any route.
 @application.route("/", defaults={"path": ""})
 @application.route("/<path:path>")
-@requires_auth
+#@requires_auth
```
Comment (ttung) [inline src/py/aspen/app.py]: is this change intentional?

Comment (phoenixAja) [reply src/py/aspen/app.py]: no i meant to remove that, since it doesn't fit in with the v0 user flow

**Review by ttung — APPROVED** [review]

**Review by lvreynoso — CHANGES_REQUESTED** [review]
Can we bring the state up to the level of `App`? We don't want state that will be shared by the entire app to be stuck in the `NavBar` component.

**Comment by phoenixAja** [comment]
@lvreynoso moved the state up to App

**Review by lvreynoso — APPROVED** [review]
Thanks Phoenix!
